app/helpers/preview_helper.rb

```diff
@@ -60,8 +60,6 @@ def prism_language_name_by_template(template:)
   def prism_language_name_by_template_path(template_file_path:)
     language = template_file_path.gsub(".html", "").split(".").last
 
-    return FALLBACK_LANGUAGE unless AVAILABLE_PRISM_LANGUAGES.include? language
-
-    language
+    render "view_components/preview_source"
```

Comment:
Did you mean to make this change?

Reply:
Nope this was a merge conflict issue. Woops!